hivemq-edge/build.gradle.kts

```diff
@@ -109,114 +110,112 @@ repositories {
-    implementation("com.fasterxml.jackson.core:jackson-databind:${property("jackson.version")}")
-    implementation("org.jctools:jctools-core:${property("jctools.version")}")
+    implementation(libs.zeroAllocationHashing)
+    implementation(libs.jctools)
 
     //mqtt-sn codec
-    implementation("com.github.simon622.mqtt-sn:mqtt-sn-codec:838f51d691")
+    implementation(libs.mqtt.sn)
+    implementation(libs.hivemq.mqttClient)
 
     //JAX-RS + Http Connector + Serializers
-    implementation("org.glassfish.jersey.containers:jersey-container-jdk-http:${property("jersey.jaxrs.sun.version")}")
-    implementation("org.glassfish.jersey.inject:jersey-hk2:${property("jersey.jaxrs.sun.version")}")
-    implementation("org.glassfish.jersey.media:jersey-media-json-jackson:${property("jersey.jaxrs.sun.version")}")
-    implementation("org.glassfish.jersey.media:jersey-media-multipart:${property("jersey.jaxrs.sun.version")}")
-    implementation("org.glassfish.jersey.inject:jersey-hk2:${property("jersey.jaxrs.sun.version")}")
-    implementation("com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:${property("jackson.version")}")
-    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${property("jackson.version")}")
+    implementation(libs.glassfish.jersey.container.jdk.http)
+    implementation(libs.glassfish.jersey.hk2)
+    implementation(libs.glassfish.jersey.media.json.jackson)
+    implementation(libs.glassfish.jersey.media.multipart)
 
-    implementation("com.hivemq:hivemq-mqtt-client:1.3.3")
+    // Jackson
+    implementation(libs.jackson.jaxrs.json.provider)
+    implementation(libs.jackson.datatype.jsr310)
+    implementation(libs.jackson.databind)
 
     //Open API
-    implementation("io.swagger.core.v3:swagger-annotations:${property("swagger.openapi.annotations.version")}")
+    implementation(libs.swagger.annotations)
 
     //JWT
-    implementation("org.bitbucket.b_c:jose4j:${property("jose4j.version")}")
-
-    // crypto
-    implementation("org.bouncycastle:bcpkix-jdk15on")
-    implementation("org.bouncycastle:bcprov-jdk15on")
+    implementation(libs.jose4j)
 
     //json schema
-    implementation("com.networknt:json-schema-validator:1.0.88")
-    implementation("com.github.victools:jsonschema-generator:4.35.0")
-    implementation("com.github.victools:jsonschema-module-jackson:4.35.0")
+    implementation(libs.jsonSchemaValidator)
+    implementation(libs.victools.jsonschema.generator)
+    implementation(libs.victools.jsonschema.jackson)
 }
 
 /* ******************** test ******************** */
 
 dependencies {
-    testAnnotationProcessor("com.google.dagger:dagger-compiler:${property("dagger.version")}")
-    testImplementation(platform("org.junit:junit-bom:${property("junit.jupiter.version")}"))
-    testImplementation("junit:junit:${property("junit.version")}")
-    testImplementation("org.junit.jupiter:junit-jupiter")
-    testRuntimeOnly("org.junit.vintage:junit-vintage-engine")
```

Comment:
I don't see the vintage engine on the right side

Reply:
re-added